<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boardContainorWindow</class>
 <widget class="QMainWindow" name="boardContainorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>370</y>
      <width>611</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>当前模式</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdBlackMu">
       <property name="styleSheet">
        <string notr="true">QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>白棋目数</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelCurModel">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdCurTotalStep">
       <property name="styleSheet">
        <string notr="true">QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="lcdWhiteMu">
       <property name="styleSheet">
        <string notr="true">QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>当前步数</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdBlackTizi">
       <property name="styleSheet">
        <string notr="true">QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdWhiteTizi">
       <property name="styleSheet">
        <string notr="true">QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>当前规则</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_20">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>黑棋目数</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelGuize">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>让9子/贴目/</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelTip">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#E8E8E8&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>未开始/劫/当前模式/坐标</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>对局提示</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>白棋提子</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>黑棋提子</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>268</width>
      <height>931</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: 9pt &quot;楷体&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}


QWidget { 
    border: 1px solid &quot;#778899F&quot;; 
    background-color: &quot;#C1CDCD&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>       注释</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;新宋体&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'新宋体'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>0</y>
      <width>616</width>
      <height>363</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btnStart">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnDeducation">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>推算模式</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btnHuiqi">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>悔棋</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnStopDeducation">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>结束推算</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="2" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2" rowspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnStop">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>停手</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="btnExitToSign">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>退出标记</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>结束</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="2">
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>152</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnToSign">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 9pt &quot;华文行楷&quot;;
	font: 9pt &quot;隶书&quot;;
    border-radius: 4px; 
    border: none; 
    width: 35px; 
    height: 35px; 
    color: rgb(28, 28, 28);
    font: bold 25px;
}

QPushButton:hover { 
    background: rgb(150, 150, 150); 
}
QPushButton:pressed{ 
    background: rgb(185, 185, 185); 
}

QWidget { 
    border: 1px solid &quot;#4F4F4F&quot;; 
    background-color: &quot;#EEE5DE&quot;; 
}
</string>
       </property>
       <property name="text">
        <string>标记点目</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
